presentation/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="margin14">14dp</dimen>
+    <dimen name="maargin20">20dp</dimen>
```

Comment:
Misspelled `maargin`, also, please add a underscore before the number.

Reply:
I suggest use  this convention:
```
<dimen name="dp_14">14dp</dimen>
<dimen name="dp_20">20dp</dimen>
```